packages/cohort_sdk/src/model/mod.rs

```diff
@@ -42,19 +29,21 @@ pub struct ResponseMetadata {
     pub duration_ms: u64,
 }
 
-#[derive(strum::Display)]
+#[derive(strum::Display, Debug)]
 // this is napi friendly copy of talos_agent::agent::errors::AgentErrorKind
 pub enum ClientErrorKind {
     Certification,
     CertificationTimeout,
-    ClientAborted,
+    SnapshotTimeout,
```

Comment:
Is this value used anywhere? Three timeouts here look suspicious :)

Reply:
updated. Thanks for spotting this.